### PR TITLE
Add fileInput widget

### LINKED ISSUE #294: Add fileInput widget
With semantic the default Shiny fileInput is working but the styling is broken. There is no other way to include fileInput. It would be ok to make default one looking good.

### PR DESCRIPTION
**Pull request by ashbaldry**
Using Fomantic-UI to enable fileInput. Resolves #294

**DoD**

- [x] Major project work has a corresponding task. If there’s no task for what you are doing, create it. Each task needs to be well defined and described.

- [x] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.

- [x] No new error or warning messages are introduced.

- [x] All interaction with a semantic functions, examples and docs are written from the perspective of the person using or receiving it. They are understandable and helpful to this person.

- [x] If the change affects code or repo sctructure, README, documentation and code comments should be updated. 

- [x] All code has been peer-reviewed before merging into any main branch.

- [x] All changes have been merged into the main branch we use for development (develop).

- [x] Continuous integration checks (linter, unit tests) are configured and passed.

- [x] Unit tests added for all new or changed logic.

- [x] All task requirements satisfied. The reviewer is responsible to verify each aspect of the task.

- [x] Any added or touched code follows our style-guide.

### CODE REVIEW

File: examples/shiny_syntax/fileInput.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- shinyUI(
+  semanticPage(
+    fileInput("file_ex", "File Input", width = "400px", type = "small"),# numericinput specific params
```
Comment (dokato) [inline examples/shiny_syntax/fileInput.R:6]: this comment should not be here `# numericinput specific params`?

**Review by dokato — APPROVED** [review]
hey @ashbaldry , perfect timing! I was about to release some patch to the latest `shiny.semantic` release on CRAN. It'd be great to include this in it.
Could make this one minor fix before merge?
Thanks a lot again 👍

File: examples/shiny_syntax/fileInput.R (hunk 1 above)
Comment (ashbaldry) [reply examples/shiny_syntax/fileInput.R]: Done

**Comment by dokato** [comment]
Perfect, thanks again @ashbaldry!

**Pull request merged**